site/config.toml

```diff
@@ -66,6 +66,9 @@ baseName = "_redirects"
 [outputs]
 home = ["HTML", "REDIRECTS"]
 
+[params]
+latest = "v0.10.0"
```

Comment:
nit: s/latest/stable ?
latest in go conjures up @latest, which we are explicitly not using.

Reply:
Sure. Updated 😃